community/index.md

```diff
@@ -8,6 +8,7 @@ title: Get in touch!
     <p>We'd love to have your feedback, suggestions, and will be happy to help with anything you need. Just get in touch!</p>
 
     <ul>
+      <li><a href="https://lists.apache.org/list.html?jclouds.apache.org">Browse and participate</a> in our mailing lists</li>
```

Comment:
> let me explain the purpose of this change

Thanks for the useful and interesting explanation - is there any way we can actually include that in the PR in some way, i.e. make it visible to the user somehow?

I feel that what you wrote is information that would be very useful to the users, too; perhaps there is an easy-ish way to explain the purpose of the various lists somewhere in more detail _before_ sending users to this page? Or perhaps there is a view that shows _all_ activity (across lists), so that we don't need to pick one and hope that it's the "right one"?